ShowerShow/Repository/UserRepository.cs

```diff
@@ -171,14 +171,5 @@ public async Task<IEnumerable<GetUserDTO>> GetUsersByName(string userName)
             return dtos;
          
         }
-
-        public async Task<bool> CheckIfUserExist(Guid userId)
```

Comment:
This method was being used bro why did you remove it?

Reply:
For deactivating/activating account it is needed: See -> UserController/ DeactivateAccount endpoint

Reply:
Ohhh I think you just moved it nvm